packages/two-factor/src/two-factor.ts

```diff
@@ -50,7 +33,7 @@ export class TwoFactor {
       throw new Error(errors.codeRequired);
     }
 
-    const twoFactorService = this.getUserService(user);
+    const twoFactorService = getUserTwoFactorService(user);
```

Comment:
Do we use the data from the twoFactor service? Maybe change `getUserTwoFactorService` to `hasTwoFactorService(user: User): boolean`

Reply:
okay 👍